Automattic/wp-calypso
# P2 Invite Links: add viewer role option

Fixes gh-1020-p2

#### Changes proposed in this Pull Request

* Add `Viewer` role option to Invite Links

#### Testing instructions
1. `svn up` your sandbox to get the recently deployed wpcom part of this fix.
2. Go to Invite People and disable existing invite links if there are any.
3. Generate new ones.
4. You should now see an invite link for the Viewer role. (Or Follower if the site is public.)
5. Try the Viewer invite link in Incognito. It should display an accept-invite page about following your site.
